api/src/main/java/org/pmiops/workbench/db/model/DbUser.java

```diff
@@ -96,10 +96,6 @@
 
   private Timestamp computeSecuritySuspendedUntil;
 
-  // potentially obsolete access module fields.  These are likely to be deleted in the near future.
-  // Moodle badges are indexed by username, not this value.  See ComplianceService.
-  @Deprecated private Integer moodleId;
```

Comment:
We don't use this field for Moodle V2 - and we've been on V2 for quite a long time.